java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -274,4 +278,20 @@ public void info_with_multi_node_route_and_options() {
             }
         }
     }
+
+    @Test
+    @SneakyThrows
+    public void config_reset_stat() {
```

Comment:
missing the config rewrite tests.

Reply:
> no IT for `CONFIG REWRITE`, because redis in IT runs without config file

The command fails

Reply:
have you seen how the other wrappers implemented these tests?